packages/components/dashboard/details-card/details-card.scss

```diff
@@ -0,0 +1,28 @@
+.details-card-title {
+  font-weight: var(--pf-global--FontWeight--normal);
+  font-size: var(--pf-global--FontSize--lg);
+}
+
+.pf-c-card__header .pf-c-card__title {
+  padding-left: var(--pf-c-card--child--PaddingLeft);
+}
+
+.details-card-body {
+  display: flex;
+}
+
+.details-card-item {
+  width: 33%;
+}
+
+.co-status-card__health-body {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: var(--pf-c-card--child--PaddingLeft);
+  padding-right: var(--pf-c-card--child--PaddingRight);
+  padding-bottom: 0;
+}
```

Comment:
```suggestion
```
move common classes to dashboard.scss...

Reply:
I think we can't, want to override the padding-bottom property only for the details card component.

Reply:
there is ` dashboard.scss` file in same directory... I am asking to remove it from here and move there... because this css is used in multiple files...

Reply:
yes then it is making changes in the status card also, I want to override only in the details card